WalkingChallenge/EventsViewController.swift

```diff
@@ -143,8 +144,10 @@ fileprivate class PastEventsDataSource: NSObject, UITableViewDataSource {
 fileprivate class EventTableDelegate: NSObject, UITableViewDelegate {
   func tableView(_ tableView: UITableView,
                  heightForRowAt indexPath: IndexPath) -> CGFloat {
-    // FIXME(compnerd) calculate this somehow?
-    return 112.0
+    return UITableViewAutomaticDimension
+  }
+  func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
```

Comment:
Would be nice to have a newline between the functions, and to wrap the arguments (80-col :-D)